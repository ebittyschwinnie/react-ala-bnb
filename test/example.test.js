describe('example', () => {
  it('can run a test', () => {
    expect(1 + 2).toMatchSnapshot();
  });
});
