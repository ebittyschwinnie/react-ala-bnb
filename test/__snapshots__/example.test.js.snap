// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example can run a test 1`] = `3`;
